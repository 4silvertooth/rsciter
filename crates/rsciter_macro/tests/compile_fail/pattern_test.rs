#[rsciter_macro::xmod]
mod with_pattern {
    pub fn test( &(x, y): &(i32, i32)) {
    }
}

fn main() {}