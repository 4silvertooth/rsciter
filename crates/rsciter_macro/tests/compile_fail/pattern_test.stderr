error: #[rsciter::xmod] Only simple parameter variable bindings are supported!
 --> tests/compile_fail/pattern_test.rs:3:18
  |
3 |     pub fn test( &(x, y): &(i32, i32)) {
  |                  ^
